CHANGELOG

```diff
@@ -1,3 +1,8 @@
+== Version 4.4.0
```

Comment:
because it changes the public API? VS being a non-public facing change that `_._.X` denotes?